numpy/random/tests/test_smoke.py

```diff
@@ -843,52 +843,3 @@ def test_fallback(self):
         e2 = entropy.random_entropy(source='fallback')
         assert_((e1 != e2))
 
-
-class TestPCG64(RNG):
```

Comment:
This is likely redundant.  Was the first "test" code.  

Ultimately this file should have the good tests saved and its name changed since I don't think smoke tests are actually needed anymore.